aiida_quantumespresso/parsers/pp.py

```diff
@@ -335,10 +335,15 @@ def parse_gaussian(self, data_file_str):
         """
         lines = data_file_str.splitlines()
 
+        title = lines[0]
+        comment = lines[1]
```

Comment:
are these variables used at all?

Reply:
They aren't, thanks. I thought prospector should catch this kind of issues, though.